Resources/Locale/en-US/components/atmos/gas-analyzer-window-component.ftl

```diff
@@ -0,0 +1,7 @@
+gas-analyzer-window-name = Gas Analyzer
```

Comment:
Files like this one should be combined.
i.e. there should be a single `gas-analyzer-component.ftl`, not one specific to the comp and one specific to the window.

You can still break up/categorise sections within the same document using `##` comments, e.g.
(Specifically two hashes, not 1 or 3, 1 is for comments attached to a message (direct line above) and 3 is for file-level commentary)

```
## Entity

gas-analyzer-component-...

## UI

gas-analyzer-window-...
```

So if you could combine all similar `x.ftl` and `x-window.ftl` files you've made into a single file, that'd be great.

Reply:
yes, there are a few instances when I wasn't sure if I should split it into multiple files or keep it as one. Luckily this is trivial to change, just some Ctrl X Ctrl V